packages/dashboard/src/components/tooltip/tooltip.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
```

Comment:
this tooltip is specific for property panel right ? so worth naming it appropriately or making another PropertyPanelToolTip which uses this ?